src/main/java/jpa/domain/line/Line.java

```diff
@@ -0,0 +1,38 @@
+package jpa.domain.line;
+
+import jpa.domain.base.BaseTimeEntity;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class Line extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
```

Comment:
이 부분은 DB schema 상으로 unique 제약 조건이 걸려있던 것 같은데, 아래와 같이 추가해주셔야 할 것 같습니다!

```suggestion
    @Column(name = "name", unique=true)
```

Reply:
`제약조건`을 빼먹었네요. 왜 이걸 확인 못했는지..
수정하겠습니다. 꼼꼼한 리뷰 감사합니다 👍